event.go

```diff
@@ -86,6 +86,13 @@ func (eb *EventBuilder) SetUnsigned(unsigned interface{}) (err error) {
 	return
 }
 
+// A VersionedEvent is a wrapper around an Event that contains information
+// about the room version.
+type VersionedEvent struct {
```

Comment:
Can we get rid of this? We have `HeaderedEvent`.

Reply:
Whoops, forgot to delete this when I added `headeredevent.go`. Thanks for pointing it out.